.travis.yml

```diff
@@ -0,0 +1,18 @@
+language: go
+
+go:
```

Comment:
Do we really want to limit testing to 1.10.x? We might as well throw it at 1.7+ since it costs us nothing and allows us more backwards capability (I currently develop on 1.7.x)